tests/unit/util/test_cubes.py

```diff
@@ -0,0 +1,27 @@
+"""
+Unit tests for cubes.py
+"""
+
+import os
+import pytest
+import iris
+import xarray as xr
+import pandas as pd
+
+from clean_air.util import cubes as cubes_module
+
+@pytest.fixture(scope="class")
+def aircraft_cube(sampledir):
+    path = os.path.join(sampledir, "aircraft", "M285_sample.nc")
+    return iris.load_cube(path, 'NO2_concentration_ug_m3')
+
+@pytest.fixture(scope="class")
+def track_dataframe(aircraft_cube):
+	ds = xr.DataArray.from_iris(aircraft_cube)
+	return ds.to_dataframe()
+
+class TestExtract:
+
+	def test_extract_series(self, aircraft_cube, track_dataframe):
```

Comment:
I would like to see descriptions of class and test in here please, but other than that very good.